style2.css

```diff
@@ -0,0 +1,217 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.text-heading {
+  font-size: 2.4rem;
+  font-weight: 400;
+}
+
+.text-heading-bold {
+  font-size: 2.4rem;
+  font-weight: 600;
+}
+
+.text-md-bold {
+  font-size: 1.6rem;
+  font-weight: 600;
+}
+
+.text-md {
+  font-size: 1.6rem;
+  font-weight: 400;
+}
+
+html {
+  font-size: 62.5%;
+  font-family: sans-serif;
+}
+
+.container > * {
+  border: 2px dashed lightgrey;
+  border-radius: 10px;
+  min-height: 100vh;
+}
+
+.three-column-grid__expand-two {
+  display: grid;
+  grid-template-columns: auto 1fr auto;
+}
+
+.three-row-grid__expand-two {
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+}
+
+.two-column-grid-expand-two {
+  grid-template-columns: 10rem 1fr;
+}
```

Comment:
I like how the layouts are all in one section, makes the code look clean and easy to read.